File: api/v2010/queue/update-example-1/update-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

queue = @client.queues('QU32a3c49700934481addd5ce1659f04d2')
               .update(max_size: 123)

puts queue.friendly_name

File: api/v2010/queue/update-example-1/update-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$queue = $twilio->queues("QU32a3c49700934481addd5ce1659f04d2")
                ->update(array("maxSize" => 123));

print($queue->friendlyName);